zipline/gens/brokers/broker.py

```diff
@@ -42,15 +42,15 @@ def time_skew(self):
         pass
 
     @abstractmethod
-    def order(self, asset, amount, limit_price, stop_price, style):
+    def order(self, asset, amount, style):
         pass
 
-    @abstractmethod
-    def get_open_orders(self, asset):
+    @abstractproperty
+    def orders(self):
         pass
```

Comment:
The same here

Reply:
My explanation above also applies here. Blotter needs to know all the orders, not just the open ones.